functions-and-scope.js

```diff
@@ -82,3 +135,19 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // highestGrade(grades) geeft 9
 // highestGrade([6, 4, 5]) geeft 6
 // highestGrade([8, 9, 4, 6, 10]) geeft 10
+
+function highestGrade(list3a) {
+    let counter3a = list3a[0];
+    let maxNumber = list3a[0];
```

Comment:
Wat goed dat je met de eerste waarde van de array begint in plaats van `0` bijvoorbeeld 👍